jaw-jdk9 needs java 9, but only module-info needs it

---
 wrapper/Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/wrapper/Makefile.am
+++ b/wrapper/Makefile.am
@@ -19,7 +19,7 @@ EXTRA_DIST = $(properties_DATA) \
 if MODULAR_JAR
 
 module-info.class : $(srcdir)/module-info.java
-	$(JAVAC) -cp $(top_builddir)/wrapper -sourcepath $(top_srcdir)/wrapper:$(top_builddir)/wrapper $(JAVACFLAGS) -d $(top_builddir)/wrapper $<
+	$(JAVAC) -cp $(top_builddir)/wrapper -sourcepath $(top_srcdir)/wrapper:$(top_builddir)/wrapper $(JAVACFLAGS) -source 9 -target 9 -d $(top_builddir)/wrapper $<
 
 $(JARFILES) : $(DEP_CLASSES) module-info.class
 	$(JAR) cfm $(JARFILES) $(srcdir)/manifest.txt org/GNOME/Accessibility/*.class module-info.class -C $(srcdir) META-INF/services/javax.accessibility.AccessibilityProvider
